poc-tests/WeatherForecastTests.cs

```diff
@@ -0,0 +1,11 @@
+namespace poc_tests
+{
+    public class WeatherForecastTests
+    {
+        [Fact]
+        public void GetWeatherForecasts_Success()
+        {
+
+        }
```

Comment:
Is this file supposed to be part of this PR?

Reply:
No. Removed this file.